requirements.txt

```diff
@@ -5,4 +5,5 @@ qiskit-ibm-experiment>=0.3.4
 matplotlib>=3.4
 uncertainties
 lmfit
-rustworkx
\ No newline at end of file
+rustworkx
+pandas>=1.1.5
```

Comment:
Now that pandas is a requirement, the `requires_pandas()` decorator can be removed and so can the install pandas note in `tutorials/calibrations.rst`.

Reply:
Seems like the decorator is not used in the rst file, but text is updated and the decorator is removed from the entire package. [46d350e](https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1253/commits/46d350e224823761b3711d24794e77bd5025b15b)